content/src/fieldConfig/login-support-page.json

```diff
@@ -0,0 +1,35 @@
+{
+  "loginSupportPage": {
+    "pageTitle": "Having trouble logging in to Business.NJ.gov?",
+    "lastUpdated": "Nov 5, 2025",
+    "multipleAccountsCallout": ":::miniCallout{ calloutType=\"informational\" }\n**If you have multiple myNewJersey accounts, use the email linked to your business.**\n:::",
+    "steps": [
+      {
+        "content": "You need a myNewJersey account to log in to [account.Business.NJ.gov](/). myNewJersey is a platform used by different government agencies in New Jersey.\n\nTo link your myNewJersey account to account.business.nj.gov:\n\n1. Make sure you're logged out of all other MyNJ accounts across your browser.\n2. Open an incognito browser.\n3. Go to [https://account.business.nj.gov/](/) and click on the \"Get Started\" button:",
+        "imagePath": "/img/login-support-home-login.png",
+        "imageAlt": "Screenshot showing the Get Started button on the Business.NJ.gov homepage"
+      },
+      {
+        "content": "4. Answer the \"Tell us about your business\" question and click \"Next.\"\n5. Once you're on the dashboard, click on the link in the purple card on the right side of your screen.",
```

Comment:
Is there any benefit to removing the step numbers from the copy and rendering them on the page as part of an ordered list?

I could see maintaining this being a little easier if steps needed to change for some reason. Also not sure if using an ordered list benefits screen-reader users.